### PR TITLE
Kusama: remove migrations applied by `spec_version: 1_000_000`

### PR DESCRIPTION
**Pull request by acatangiu**
Runtimes 1.0.0 has been released: https://github.com/polkadot-fellows/runtimes/releases/tag/v1.0.0 and Kusama has now successfully updated on-chain to spec: `1_000_000`.

This PR removes migrations that have been applied by the successful update of the Kusama runtime to above version.

### CODE REVIEW

**Comment by acatangiu** [comment]
The 1st commit is the mandatory one, it removes the `SessionKeys` migration which doesn't have protections in place to allow multiple runs of the migration.

The other ones' removal at this point is optional, but if we see them applied on-chain, let's also remove them.

File: relay/kusama/src/lib.rs
Comment (bkchr) [inline relay/kusama/src/lib.rs:1626]:
```suggestion
```

Can also be removed.

**Review by bkchr — APPROVED** [review]

**Review by liamaharon — APPROVED** [review]
Why only remove from Kusama?

**Comment by acatangiu** [comment]
> Why only remove from Kusama?

I'm thinking we do separate PRs that follow actual on-chain upgrades + migrations for each chain.

When kusama is upgraded and we can **verify** these migrations have been successful we'll merge PR to remove them.

My suggestions is same process, different PR+timeline for Polkadot.

**Comment by acatangiu** [comment]
referenda for upgrading Kusama: https://kusama.subsquare.io/referenda/297

**Comment by acatangiu** [comment]
Kusama now successfully updated to `spec: 1_000_000` - all these migrations have now run (and been finalized) on-chain so we can remove them.

Please note, we **have to remove** the session keys migration before the next release as it doesn't gracefully handle reapplying it.

**Comment by acatangiu** [comment]
@andresilva please a rank-3 review here to get this in and avoid accidents like on Westend

File: relay/kusama/src/paras_scheduler_migration.rs
Comment (acatangiu) [inline relay/kusama/src/paras_scheduler_migration.rs:109]: Instead of this, I can also remove this file completely, but thought it might be useful for future reference.. lemme know if you think otherwise.

Comment (ordian) [reply relay/kusama/src/paras_scheduler_migration.rs:109]: I'd just remove it. We have git history for future reference.

**Review by ordian — APPROVED** [review]

**Review by andresilva — APPROVED** [review]

**Comment by acatangiu** [comment]
hmm.. since this touches `relay/` files, it needs 4 rank 3 approvals

fellows, please review!

**Comment by acatangiu** [comment]
I just rebased and fixed conflicts. Someone with merge rights, pls take a look and merge before we keep adding new stuff and start confusing already applied migrations with incoming new ones.

**Comment by acatangiu** [comment]
> 6 of 13 checks passed 

this got auto-merged after rebase without running all CI tests - any way we can see if these failed or simply didn't run or what? 😃

**Comment by andresilva** [comment]
You can check the CI build on master after this got merged.

**Pull request merged**
